docs/INTRO-cmake.md

```diff
@@ -37,19 +37,17 @@ cmake --build build
 ```
 
 # Run:
-## Windows (Visual Studio Generator)
-The executable should be in the `build/Debug` directory:
+The executable should be in the `build` directory:
+
 ```sh
-cd build/Debug
+cd build
 ./hello
 ```
 
-> NOTE: If there wasn't an executable here despite the above Build section running successfully, it's likely because you're following this guide using a different toolchain. This section is written with the assumption that you both had a Visual Studio toolchain installed, and that CMake selected to configure it's build for it. Please take a look just below.
+If there wasn't an executable there despite the above Build section running successfully, it's likely because you're following this guide using the Visual Studio toolchain, it should instead be in the `build/Debug` directory.
```

Comment:
```suggestion
If there wasn't an executable there despite the above Build section running successfully, it's likely because you're following this guide using the Visual Studio toolchain, it should instead be in the `build/Debug` directory:
```
(and remove the following blank line)